test/unit/presenters/publishing_api_presenters/coming_soon_test.rb

```diff
@@ -6,6 +6,7 @@ class PublishingApiPresenters::ComingSoonTest < ActiveSupport::TestCase
     @locale = 'en'
     @publish_timestamp = 1.day.from_now
     @content_id = SecureRandom.uuid
+    SecureRandom.stubs(:uuid).returns(@content_id)
```

Comment:
I tend to favour `stubs(uuid: @content_id)` which seems somewhat more concise and still as intention-revealing. At least to my eyes, anyway :smile: